spec/dish_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "dish"
+
+describe Dish do
+  let(:dish) { Dish.new }
+
+  it "creates a instance of Dish" do
+    expect(dish).to be_instance_of(Dish)
+  end
+
+  # it "initializes with a hash of dishes " do
```

Comment:
Nice I can see where you are going with this.

We can be explicit here and say:

``` expect(dish.dishes).to eq({"pizza":5, "coke":2})